File: 462-minimum-moves-to-equal-array-elements-ii/minimum-moves-to-equal-array-elements-ii.java
class Solution {
    public int minMoves2(int[] nums) {
        var n = nums.length;
        Arrays.sort(nums);

        var cnt = 0;
        var mid = nums[n / 2];

        for (var num : nums)
            cnt += Math.abs(num - mid);

        return cnt;
    }
}

File: 462-minimum-moves-to-equal-array-elements-ii/minimum-moves-to-equal-array-elements-ii.py
class Solution:
    def minMoves2(self, nums: List[int]) -> int:
        nums.sort()
        n = len(nums)

        cnt = 0

        mid_element = nums[n // 2]
        for num in nums:
            cnt += abs(num - mid_element)

        return cnt
